lib/runner/extensions/event.command.js

```diff
@@ -11,8 +11,9 @@ var _ = require('lodash'),
     createItemContext = require('../create-item-context'),
 
     ASSERTION_FAILURE = 'AssertionFailure',
-    SAFE_CONTEXT_VARIABLES = ['_variables', 'environment', 'globals', 'collectionVariables', 'cookies', 'data',
-        'request', 'response'],
+    SAFE_CONTEXT_VARIABLES =
+        ['_variables', 'environment', 'globals', 'secrets', 'collectionVariables', 'cookies', 'data',
```

Comment:
@Pranav2612000 let's call them `vaultVariables` similar to `collectionVariables`. Calling them `secrets` adds ambiguity because of the `secret` type that exists.